test/unit/android_test.rb

```diff
@@ -6,92 +6,91 @@ class AndroidTest < Minitest::Test
+    @browser.ua = Browser['ANDROID_ECLAIR_21']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(2.1)
   end
 
   test "detect android froyo (2.2)" do
-    @browser.ua = $ua['ANDROID_FROYO']
-    assert @browser.android?
-    assert @browser.android?(2.2)
+    @browser.ua = Browser['ANDROID_FROYO']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(2.2)
   end
 
   test "detect android gingerbread (2.3)" do
-    @browser.ua = $ua['ANDROID_GINGERBREAD']
-    assert @browser.android?
-    assert @browser.android?(2.3)
+    @browser.ua = Browser['ANDROID_GINGERBREAD']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(2.3)
   end
 
   test "detect android honeycomb (3.0)" do
-    @browser.ua = $ua['ANDROID_HONEYCOMB_30']
-    assert @browser.android?
-    assert @browser.android?(3.0)
+    @browser.ua = Browser['ANDROID_HONEYCOMB_30']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(3.0)
   end
 
   test "detect android ice cream sandwich (4.0)" do
-    @browser.ua = $ua['ANDROID_ICECREAM']
-    assert @browser.android?
-    assert @browser.android?(4.0)
+    @browser.ua = Browser['ANDROID_ICECREAM']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(4.0)
   end
 
   test "detect android jellybean (4.1)" do
-    @browser.ua = $ua['ANDROID_JELLYBEAN_41']
-    assert @browser.android?
-    assert @browser.android?(4.1)
+    @browser.ua = Browser['ANDROID_JELLYBEAN_41']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(4.1)
   end
 
   test "detect android jellybean (4.2)" do
-    @browser.ua = $ua['ANDROID_JELLYBEAN_42']
-    assert @browser.android?
-    assert @browser.android?(4.2)
+    @browser.ua = Browser['ANDROID_JELLYBEAN_42']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(4.2)
   end
 
   test "detect android jellybean (4.3)" do
-    @browser.ua = $ua['ANDROID_JELLYBEAN_43']
-    assert @browser.android?
-    assert @browser.android?(4.3)
+    @browser.ua = Browser['ANDROID_JELLYBEAN_43']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(4.3)
   end
 
   test "detect android kitkat (4.4)" do
-    @browser.ua = $ua['ANDROID_KITKAT']
-    assert @browser.android?
-    assert @browser.android?(4.4)
+    @browser.ua = Browser['ANDROID_KITKAT']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(4.4)
   end
 
   test "detect android lollipop (5.0)" do
-    @browser.ua = $ua['ANDROID_LOLLIPOP_50']
-    assert @browser.android?
-    assert @browser.android?(5.0)
+    @browser.ua = Browser['ANDROID_LOLLIPOP_50']
+    assert @browser.platform.android?
+    assert @browser.platform.android?(5.0)
   end
 
   test "detect android lollipop (5.1)" do
-    @browser.ua = $ua['ANDROID_LOLLIPOP_51']
-    assert @browser.android?
-    assert @browser.android?(5.1)
+    @browser.ua = Browser['ANDROID_LOLLIPOP_51']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.